python/lsst/dax/obscore/script/obscore_export.py

```diff
@@ -0,0 +1,54 @@
+# This file is part of dax_obscore.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (http://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+__all__ = ["obscore_export"]
+
+import yaml
+from lsst.daf.butler import Butler
+
+from .. import ExporterConfig, ObscoreExporter
+
+
+def obscore_export(repo: str, destination: str, config: str, format: str) -> None:
+    """Export Butler datasets as ObsCore Data Model in parquet format.
+
+    Parameters
+    ----------
+    repo : `str`
+        URI to the butler repository.
+    destination : `str`
+        Location of the output file.
+    config : `str`
+        Location of the configuration file.
+    format : `str`
+        Output format, 'csv' or 'parquet'
+    """
+    butler = Butler(repo)
+    with open(config) as stream:
+        config_data = yaml.safe_load(stream)
```

Comment:
Could use `lsst.daf.butler.Config` here (which gives you a bit more flexibility and will read from URI), and I think can still be passed to the pydantic constructor (maybe would have to convert to a real dict first)

Reply:
I did not need the extra flexibility, but reading from URL may be useful. I'll try to switch to that.

Reply:
That worked fine, did not need to convert to dict before passing to pydantic.